stacks/apps/dovetail-router.yml

```diff
@@ -795,6 +796,8 @@ Resources:
               Value: !Ref kApplicationPort
             - Name: REDIRECT_HOST
               Value: !Ref DovetailCdnHostname
+            - Name: REDIRECT_PREFIX
+              Value: !If [IsPrimaryRegion, !Ref "AWS::NoValue", !Ref DovetailCdnRedirectPrefix]
```

Comment:
Is it the primary region that has no prefix? Or us-east-1?

Reply:
The primary region (which is us-east-1).  Which more importantly, is the "Primary Region" in the CDN template.

![image](https://user-images.githubusercontent.com/1410587/189729487-64e5131c-3d48-4bd0-8b9f-df49f3d38cb7.png)

This may just be temporary.  I don't want to change too much with this deploy.  But once things are working, I may get rid of the `!If` here.

Reply:
Oh ok. We definitely shouldn't have anything in Spire that assumes PrimaryRegion always equals us-east-1. That's true now, but the moment where that becomes not true will be when we're dealing with some major issue, so not the time we want to be tracking down a weird behavior change.

Reply:
I did already set the paramstore values for `/us-east-1/` too.  So once the `!If` is gone, it should change to the `use1` prefix.

Reply:
rgr